debian/copyright

```diff
@@ -0,0 +1,62 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: novnc
+Source: https://github.com/pi-top/noVNC
+Upstream-Contact: deb-maintainers@pi-top.com
```

Comment:
maybe we should include some reference to the original source? like they did here https://salsa.debian.org/openstack-team/third-party/novnc/-/blob/debian/yoga/debian/copyright

Reply:
makes sense, I added a link to our fork but it's better to point to the original source